functions/Remove-DbaDbBackupRestoreHistory.ps1

```diff
@@ -54,75 +54,69 @@ function Remove-DbaDbBackupRestoreHistory {
     .EXAMPLE
         PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016
 
-        Remove backup and restore history on SQL Server sql2016 older than 30 days (default period)
+        Prompts for confirmation then deletes backup and restore history on SQL Server sql2016 older than 30 days (default period)
 
-        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -KeepDays 100
+        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -KeepDays 100 -Confirm:$false
 
-        Remove backup and restore history on SQL Server sql2016 older than 100 days
+        Remove backup and restore history on SQL Server sql2016 older than 100 days. Does not prompt for confirmation.
 
         PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -Database db1
 
-        Remove complete backup and restore history for database db1 on SQL Server sql2016 
+        Prompts for confirmation then deletes all backup and restore history for database db1 on SQL Server sql2016
 
         PS C:\> Get-DbaDatabase -SqlInstance sql2016 | Remove-DbaDbBackupRestoreHistory -WhatIf
 
-        Remove complete backup and restore history for all databases on SQL Server sql2016 
-#>
-    [CmdletBinding(SupportsShouldProcess)]
+        Remove complete backup and restore history for all databases on SQL Server sql2016
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
     param (
-        [parameter(Mandatory, ParameterSetName = "instance")]
```

Comment:
ParamterSets are cool in theory but we've opted against them for most commands because the resulting errors when param sets are mixed is so confusing. We handle the mix/match in the code itself `if ($sqlinstance -and -not $database)` to ensure the error message is understandable.